prompts/claude/ao-run/SKILL.md

```
@@ -0,0 +1,58 @@
+
+**Steps**
+
+1) Validate PRD exists:
+
+```bash
+test -f .agents/tasks/prd.json
+```
+
+If missing, stop and suggest `/ao-start`.
+
+2) Ensure execution happens in a dedicated worktree for the current feature branch.
```

markdownlint-cli2 (0.20.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)